openapi.yaml

```diff
@@ -2732,6 +2891,13 @@ components:
       description: Username of a user on the controller.
       schema:
         type: string
+    Wait:
+      in: query
+      name: wait
+      required: false
+      schema:
+        type: boolean
+        default: true
```

Comment:
Why do we want `Wait` to be true here? I would have thought false as with KeepInstance.

Reply:
We discussed this in person, but this is "juju remove-machine --no-wait". I was just trying to avoid the "!NoWait" structures where you start having to think in multiply inverted negatives. (Hence the default to Wait: true).
This isn't "wait for the machine to be deleted before returning" it is "wait for the hooks to be completed successfully before destroying the instance".

Reply:
We *could* make the parameter "wait-for-hooks" if we wanted to be a bit more obvious about it.

Reply:
I would make it just that.